authors/apps/articles/models.py

```diff
@@ -37,3 +37,28 @@ def save(self, *args, **kwargs):
 
     class Meta:
         ordering = ['-created_at']
+   
+
+class Comment(models.Model):
+   """
+   Model for comments
+   """
+   
+   body = models.CharField(max_length=200)
+   created_at = models.DateTimeField(auto_now_add=True)
+   updated_at = models.DateTimeField(auto_now=True)
+   author = models.ForeignKey(User, related_name='comments', on_delete=models.CASCADE)
+   article = models.ForeignKey(ArticlesModel, related_name='comments', on_delete=models.CASCADE)
+   parent = models.ForeignKey('self',
+               null=True,
+               blank=True,
+               related_name='threads',
+               on_delete=models.CASCADE
+               )
```

Comment:
the indentation for this seems a bit off to me, I'd do it like this :
```
parent = models.ForeignKey(
   'self',
    null=True,
    blank=True,
    related_name='threads',
    on_delete=models.CASCADE
)
```
I'm not super sure but this looks more readable 😅, what do you think?